src/main/java/org/opentripplanner/prediction_client/VehiclePresence.java

```diff
@@ -0,0 +1,184 @@
+package org.opentripplanner.prediction_client;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import org.locationtech.jts.geom.Envelope;
+
+import java.util.List;
+
+public class VehiclePresence {
+
+    private String vehicle;
```

Comment:
By reading this class, I couldn't determine what "vehicle" represents. Can you change name to sth more expressive?

Reply:
At the cost of annotation for deserialisation. Since it is DTO class